src/Sulu/Bundle/ContentBundle/Resources/config/serializer/Document.BasePageDocument.xml

```diff
@@ -23,9 +23,7 @@
         <property name="creator" type="integer" groups="defaultPage,preview"/>
         <property name="changer" type="integer" groups="defaultPage,preview"/>
         <property name="created" type="DateTime" groups="defaultPage,preview"/>
-        <property name="authored" groups="defaultPage,preview">
-            <type><![CDATA[DateTime<'Y-m-d'>]]></type>
```

Comment:
when the type in the database is DateTime we have to include this (maybe with an additional time in the result).

Reply:
I've simply added `type="DateTime"` which seems to serialize DateTime to a suitable format (`2017-05-01T06:14:00+0000`).

Is anything wrong with that?